### ISSUE TITLE
agc

### ISSUE THREAD

**Issue by KOLANICH** [dkim0419/SoundRecorder#9]
Agc is needed.

**Comment by fabianrost84** [comment]
AGC might already be working on some devices. Currently, the app captures audio from `MediaRecorder.AudioSource('MIC')`. According to [this answer on stackoverflow](http://stackoverflow.com/a/14380936/5142797) AGC is highly device specific.

**Comment by KOLANICH** [comment]
>AGC might already be working on some devices.
Yes. But  I meant explicit one, like the one in [ASR](https://play.google.com/store/apps/details?id=com.nll.asr)

**Comment by fabianrost84** [comment]
Also this feature would to switch from [`MediaRecorder`](https://developer.android.com/reference/android/media/MediaRecorder.html) to [`AudioRecord`](https://developer.android.com/reference/android/media/AudioRecord.html). 
Not sure if this is in agreement with @dkim0419's idea of a "simple" audio recorder.
